src/main/java/net/dv8tion/jda/api/entities/Guild.java

```diff
@@ -2584,6 +2584,91 @@ default RestAction<RichCustomEmoji> retrieveEmoji(@Nonnull CustomEmoji emoji)
     @Nonnull
     List<GuildVoiceState> getVoiceStates();
 
+    /**
+     * Load the member's voice state for the specified user.
+     * <br>If the member is already loaded it will be retrieved from {@link #getMemberById(long)} and
```

Comment:
```suggestion
     * Load the member's voice state for the specified user.
     *
     * <br>If the member is already loaded it will be retrieved from {@link #getMemberById(long)} and
```